src/Python/tests.py

```diff
@@ -94,104 +94,104 @@ def test_tracker_commons_and_units(self):
         
         # Errors because "tracker-commons":true is not present
         with self.assertRaises(jsonschema.exceptions.ValidationError):
-            WCONWorm.load(StringIO('{"tracker-blah":true, "units":{}, "data":[]}'))
+            WCONWorm.load(StringIO('{"tracker-blah":true, "units":{"t":"s","x":"mm","y":"mm"}, "data":[]}'))
         with self.assertWarns(UserWarning):
-            WCONWorm.load(StringIO('{"units":{}, "data":[]}'))
+            WCONWorm.load(StringIO('{"units":{"t":"s","x":"mm","y":"mm"}, "data":[]}'))
         # If we're being explicitly told that this is NOT a WCON file,
         # the parser should raise an error.
         with self.assertRaises(jsonschema.exceptions.ValidationError):
-            WCONWorm.load(StringIO('{"tracker-commons":false, "units":{}, "data":[]}'))
+            WCONWorm.load(StringIO('{"tracker-commons":false, "units":{"t":"s","x":"mm","y":"mm"}, "data":[]}'))
 
         # This should fail because "units" is required
         with self.assertRaises(jsonschema.exceptions.ValidationError):
             WCONWorm.load(StringIO('{"tracker-commons":true, "data":[]}'))
 
         # The smallest valid WCON file (Empty data array should be fine)
-        WCONWorm.load(StringIO('{"tracker-commons":true, "units":{}, "data":[]}'))
+        WCONWorm.load(StringIO('{"tracker-commons":true, "units":{"t":"s","x":"mm","y":"mm"}, "data":[]}'))
 
         # Duplicate keys should cause the parser to fail
         with self.assertRaises(KeyError):
             WCONWorm.load(StringIO('{"tracker-commons":true, '
-                                    '"units":{"t":"s", "t":"s"}'
+                                    '"units":{"t":"s","t":"s","x":"mm","y":"mm"}'
```

Comment:
This is intentional; I'm testing to verify that it will fail on that erroneous data.

(viz. `with self.assertRaises(KeyError):` on the previous line)